src/App.svelte

```diff
@@ -256,7 +262,7 @@
             <Input
               type="checkbox"
               checked={isFilteredSet(trait.name)}
-              label={trait.name}
```

Comment:
Hey @cariehl really appreciate the PR, I've gone ahead and tweaked your usage of forEach slightly as it doesn't concat the return values for you, and moved the code out to a function but that's more for style than functionality reasons, great idea, really makes things more clear 👍